### ISSUE TITLE
Rad patch's should propagate

### ISSUE THREAD

**Issue by slack-coder** [radicle-dev/heartwood#383]
A patch fails to propagate from a contributor to the project maintainer.

The error is related to the maintainer attempting to fetch a non-existing git commit object from the contributor:

```
2023-03-04T11:41:42.219-05:00 DEBUG service  Draining outbox for session z6MknegFQPn3LFCkpM1f1d9a3AerPcqiN8DcmCGetZxkibvh (0 message(s))
2023-03-04T11:41:42.255-05:00 DEBUG worker   Git: remote: error: Object 2eaf5835751798dce20d9805de33da09ad3b41f5 not a commit
2023-03-04T11:41:42.255-05:00 DEBUG worker   Git: remote: fatal: revision walk setup failed
```

To recreate the error, run this script from the root of this project.  Beware it will delete any existing 'tmp' directory.

https://github.com/slack-coder/heartwood/blob/slack-coder/test-patch-propagation/script/propagate-patch-test.sh

**Comment by cloudhead** [comment]
Is this related to the "corruption" issue? Otherwise it's just a matter of calling `announce-refs` after creating the patch.

**Comment by FintanH** [comment]
I wonder if it's because they're not in the signed refs @cloudhead

**Comment by cloudhead** [comment]
Yeah, this should be fixed by one of my PRs :)

**Comment by cloudhead** [comment]
Closed by #375
